libs/blocks/global-navigation/utilities/getUserEntitlements.js

```diff
@@ -75,13 +77,19 @@ const mapSubscriptionCodes = (offers) => {
         }
       });
     }
+
+    if (Object.prototype.hasOwnProperty.call(offer, 'offer_id')) {
+      const { offer_id, ...rest } = offer;
+      offers[offer_id] = rest;
```

Comment:
I just noticed a bug in the code, as we mutate the empty entitlements, on subsequent calls they still have the mutated values. Would be nice if you fix this in your PR the code is here: https://github.com/mokimo/milo/commit/6598d6340941e914f59979f68a3b596a4a392979

Otherwise I'll open a follow up PR

Reply:
Thanks a lot @mokimo ! I included this change